runtime/src/accounts_db.rs

```diff
@@ -1424,9 +1424,10 @@ impl AccountsDB {
 
     fn get_blake3_slot(operating_mode: &OperatingMode) -> Slot {
         match operating_mode {
-            OperatingMode::Development => 0,
-            // Epoch 75
-            OperatingMode::Stable => 32_400_000,
+            // Epoch 400
+            OperatingMode::Development => 3_276_800,
+            // Epoch 78
+            OperatingMode::Stable => 33_696_000,
             OperatingMode::Preview => std::u64::MAX,
```

Comment:
Testnet epoch 95?  That'll give lots of update runway there too

Reply:
Ah ok, wasn't sure when the next release would be, sounds good!